### ISSUE TITLE
First characters in some indentation are invisible

### ISSUE THREAD

**Issue by tamton-aquib** [gantoreno/nvim-gabriel#1]
Hi, firstly I just want to say this is an awesome colorscheme.

The actual issue is, the first character in some indentation levels are invisible (notice the items in first indent):
![image](https://github.com/gantoreno/nvim-gabriel/assets/77913442/2cb577cf-5303-4d0a-9f75-51f8c7e58917)

Thought it was indent-blankline plugin but I tried turning it off(`IndentBlanklineDisable`) but the problem still persist.

Additional info:
- Tried turning off treesitter highlighting too (`:TSBufDisable highlight`)
- Happens in all languages.

**Comment by gantoreno** [comment]
Hey there! First of all, thank you so much for the kind comments :)

This is a really strange one, I've used indent-blankline in the past and haven't seen this in the past. I just tried re-creating the scenario but my indents look fairly normal so far

<img width="312" alt="image" src="https://github.com/gantoreno/nvim-gabriel/assets/43397475/afc0d0fc-f467-4559-b28f-ec2dc0392aae">

There are a few things that I'm curious about your setup, which _might_ be related:

- Does the problem happen only with this colorscheme?
- Does disabling indent-blankline fix the problem?
- Do you have any custom `listchars` set in your vim config?

Also, are you using base indent-blankline or is there any custom configuration set for your editor? Not really sure what might be causing this, but it's definitely worth investigating :)

**Comment by tamton-aquib** [comment]
I swear it was happening all over before 😭 . Now its not happening much.
Also I forgot to say I was on neovide and it happens only on neovide as of now i checked.

But for the questions:
- No, it happens for all the colorshemes now (swear it wasnt happening before *sobs*).
- Tried disabling it like the issue description said.
- No listchars set. I disabled it in options.lua.

I usually dont open issues in any repos 😅. But I forgot to check with the normal terminal. And when I checked with other schemes it was working before. 

So in conclusion, it isnt this colorschemes issue. Closing the issue 👍🏻

**Comment by gantoreno** [comment]
Just another day in sofware development land, where things suddenly break and we don't really know why 😅

I missed the part where you disabled blankline! Sorry about that. Either way, hope you get this sorted out! Again, thanks for the kind comments, and let me know if there are any other things I can improve upon on this one 👍🏻
